requirements.txt

```diff
@@ -2,5 +2,5 @@ numpy==1.13.*
 pep8==1.7.*
 polymath==0.1.17
 pylint==1.7.*
-pypuf-helper==0.1.2.*
+pypuf-helper==0.1.4
```

Comment:
I believe 0.1.5 is the [current version](https://pypi.python.org/pypi/pypuf_helper/0.1.5).

Reply:
Yes but the current master branch is not int8 ready :).

Reply:
woops